misc/semver-checker/modules/semver-checker-core/src/test/resources/testcases/enumerationDeclaration/advanceEnumeration.json

```diff
@@ -0,0 +1,170 @@
+[
+  {
+    "description": "Remove enum member , Remove public qualifier",
+    "oldCode": "public enum color{\n\tRed,\n\tBlue,\n\tGreen\n}",
```

Comment:
Have little concern regarding, Can there be a scenario like below  and if there what would be the version impact of this?
scenario 
**oldCode**
```ballerina
public enum color{
    Red,
    Blue,
    Green
}
```
**newCode**
```ballerina
public enum color{
    Red = "red",
    Blue = "blue",
    Green = "Green"
}
```

Reply:
@lnash94 IMO It will be a non-breaking change since the users access enum values always using its identifier. I've added some more test cases with the [c335f2d](https://github.com/ballerina-platform/ballerina-lang/pull/37482/commits/c335f2d0911124db320fe9df36e8da432ab7b106).